SED/sed_m5_fe4.txt

```diff
@@ -0,0 +1,100 @@
+   0.10965     2.01e-26
```

Comment:
We might modify these files to give them a simple header like ' # Wavelength Intensity'